### ISSUE TITLE
Easy syntax conflict

### ISSUE THREAD

**Issue by colintedford** [glensc/dokuwiki-plugin-pageredirect#18]
I just belatedly updated from "Ponder Stibbons" to "Hrun" and my redirects using the `#redirect` syntax now show "1. redirect [pagename]" instead of redirecting. The `~~REDIRECT>` style still works, but of course my existing redirects use `#redirect` because it's easier to type.

**Comment by jayqi** [comment]
Are you perhaps using the [Creole](https://www.dokuwiki.org/plugin:creole) plugin or some other plugin that uses `#` as the syntax for an ordered list? That looks to be what is happening. I am using Creole, and I have the same syntax conflict, so I've been stuck using the `~~REDIRECT>` option.

**Comment by colintedford** [comment]
Aha, I am using the Creole plugin; thanks for dispelling the mystery.

**Comment by glensc** [comment]
looks like [parser::parse():118](https://github.com/splitbrain/dokuwiki/blob/release_stable_2016-06-26a/inc/parser/parser.php#L118-L119) prepends `\n` to document and creole matches `\n#` first, breaking `pageredirect` plugin able to match full word

so even if your document content is: `#redirect start` when document is parsed it becames `
\n#redirect start\n`

and here's creole pattern that is overly greedy: [syntax_plugin_creole_listblock::connectTo()](
https://github.com/dokufreaks/plugin-creole/blob/f36bf242f10dfb7a747183910fe9c622cf68b384/syntax/listblock.php#L30-L34)

```php
             $this->Lexer->addEntryPattern(
                '\n[ \t]*[\#\*](?!\*)',
                $mode,
                'plugin_creole_listblock'
             );
```

**Comment by glensc** [comment]
should be fixed with 36ebf4a
